tests/acceptance/fixtures/html/universal.html

```diff
@@ -3,7 +3,7 @@
         <meta charset="utf-8" />
     
         <script src="http://localhost:9999/dist/answerstemplates.compiled.min.js"></script>
-        <script src="http://localhost:9999/dist/answers.min.js"></script>
+        <script src="http://localhost:9999/dist/answers.js"></script>
         <link rel="stylesheet" type="text/css" href="http://localhost:9999/dist/answers.css">
```

Comment:
Just double checking, but does npm run dev keep the other two assets listed up to date?

Reply:
Yup! both of these files are currently output by the dev gulp task